src/components/subreddits/index.js

```diff
@@ -3,14 +3,30 @@ import React from 'react';
 import Subreddit from './item';
 
 export default class SubredditsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selected: null
+    }
+  }
+
+  onChildClick(selected) {
+    this.setState({
+      selected: selected
```

Comment:
this name `selected` can be a bit confusing because it's really the `selectedSubredditName`, right? So when we do a comparison on line 28 against display_name it makes sense but without that, someone would have to go and look at where this is being used